### PR TITLE
added few arbitrum tokens

### PR DESCRIPTION
**Pull request by darvinrio**
Brief comments on the purpose of your changes:
Needed Decimal values and Symbol for some protocol work. 

**For Dune Engine V2**

I've checked that:

### General checks:
* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: models/tokens/arbitrum/tokens_arbitrum_erc20.sql [hunk 1]
```diff
@@ -106,5 +106,13 @@ FROM (VALUES
         ,('0xda492c29d88ffe9b7cbfa6dc068c2f9befae851b', 'CUSDCLP', 18)
         ,('0xb86af5eb59a8e871bfa573fa656123ea86f47c3a', 'CWETHLP', 18)
         ,('0x1426cf37caa89628c4da2864e40cf75e6d66ac6b', 'RELAY', 18)
-        ,('0x912ce59144191c1204e64559fe8253a0e49e6548', 'ARB',  18)
+        ,('0x912ce59144191c1204e64559fe8253a0e49e6548', 'ARB', 18)
```
Comment (Hosuke) [inline models/tokens/arbitrum/tokens_arbitrum_erc20.sql:109]: https://arbiscan.io/token/0x912ce59144191c1204e64559fe8253a0e49e6548

File: models/tokens/arbitrum/tokens_arbitrum_erc20.sql [hunk 2]
```diff
@@ -106,5 +106,13 @@ FROM (VALUES
...
+        ,('0x3d9907f9a368ad0a51be60f7da3b97cf940982d8', 'GRAIL', 18)
```
Comment (Hosuke) [inline models/tokens/arbitrum/tokens_arbitrum_erc20.sql:110]: https://arbiscan.io/token/0x3d9907f9a368ad0a51be60f7da3b97cf940982d8

File: models/tokens/arbitrum/tokens_arbitrum_erc20.sql [hunk 3]
```diff
@@ -106,5 +106,13 @@ FROM (VALUES
...
+        ,('0xb40dbbb7931cfef8be73aeec6c67d3809bd4600b', 'PPO', 18)
+        ,('0x49e050df648e9477c7545fe1779b940f879b787a', 'esGMD', 18)
```
Comment (Hosuke) [inline models/tokens/arbitrum/tokens_arbitrum_erc20.sql:112]: https://arbiscan.io/token/0x49e050df648e9477c7545fe1779b940f879b787a

File: models/tokens/arbitrum/tokens_arbitrum_erc20.sql [hunk 4]
```diff
@@ -106,5 +106,13 @@ FROM (VALUES
...
+        ,('0x5d59e5837f7e5d0f710178eda34d9ecf069b36d2', 'Y2K(Dummy)', 18)
```
Comment (Hosuke) [inline models/tokens/arbitrum/tokens_arbitrum_erc20.sql:113]: https://arbiscan.io/token/0x5d59e5837f7e5d0f710178eda34d9ecf069b36d2

File: models/tokens/arbitrum/tokens_arbitrum_erc20.sql [hunk 5]
```diff
@@ -106,5 +106,13 @@ FROM (VALUES
...
+        ,('0x0c880f6761f1af8d9aa9c466984b80dab9a8c9e8', 'PENDLE', 18)
```
Comment (Hosuke) [inline models/tokens/arbitrum/tokens_arbitrum_erc20.sql:114]: https://arbiscan.io/token/0x0c880f6761f1af8d9aa9c466984b80dab9a8c9e8

File: models/tokens/arbitrum/tokens_arbitrum_erc20.sql [hunk 6]
```diff
@@ -106,5 +106,13 @@ FROM (VALUES
...
+        ,('0x07e49d5de43dda6162fa28d24d5935c151875283', 'GOVI', 18)
```
Comment (Hosuke) [inline models/tokens/arbitrum/tokens_arbitrum_erc20.sql:115]: https://arbiscan.io/token/0x07e49d5de43dda6162fa28d24d5935c151875283

File: models/tokens/arbitrum/tokens_arbitrum_erc20.sql [hunk 7]
```diff
@@ -106,5 +106,13 @@ FROM (VALUES
...
+        ,('0x65c936f008bc34fe819bce9fa5afd9dc2d49977f', 'Y2K', 18)
+        ,('0xda51015b73ce11f77a115bb1b8a7049e02ddecf0', 'NEU', 18)
```
Comment (Hosuke) [inline models/tokens/arbitrum/tokens_arbitrum_erc20.sql:117]: https://arbiscan.io/token/0xda51015b73ce11f77a115bb1b8a7049e02ddecf0

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank youn @darvinrio

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**